### ISSUE TITLE
Combine the loss layers

### ISSUE THREAD

**Issue by 0x00b1** [broadinstitute/keras-rcnn#107]
keras-rcnn has two region proposal network (RPN) loss layers (`RPNClassification` and `RPNRegression`) and two region-based convolutional neural network (RCNN) layers (`RCNNClassification` and `RCNNRegression`). I think it makes sense to combine the two RPN layers into one RPN layer and the two RCNN layers into one RCNN layer in an effort to further simplify the keras-rcnn’s public API.

**Comment by hgaiser** [comment]
So there'd be one layer computing the RPN loss and one layer to compute the RCNN loss? I like that idea!
